web/src/features/fireWeather/components/tables/ComparisonTableRow.tsx

```diff
@@ -93,24 +96,43 @@ const useStyles = makeStyles({
   }
 })
 
-const formatWindSpeedDirection = (
-  source: NoonForecastValue | ObservedValue | ModelValue | undefined,
+const formatWindSpeedForecast = (
+  source: NoonForecastValue | undefined,
   valueClassName: string[]
 ): ReactElement => {
   return (
     <div>
       {typeof source?.wind_speed === 'number' && (
         <div className={valueClassName[0]}>
-          {source?.wind_speed?.toFixed(WIND_SPEED_VALUES_DECIMAL)} km/h
+          {formatForecastWindSpeedValue(source.wind_speed)}
         </div>
       )}
-      {typeof source?.wind_speed === 'number' &&
-        typeof source?.wind_direction === 'number' &&
-        ' '}
+    </div>
+  )
+}
+
+const formatWindSpeed = (
+  source: ObservedValue | ModelValue | undefined,
+  valueClassName: string[]
+): ReactElement => {
+  return (
+    <div>
+      {typeof source?.wind_speed === 'number' && (
```

Comment:
Can you simplify to remove the type check?

`formatWindSpeedValue(source?.wind_speed)}` should just work for undefined values.

Reply:
Depends where one does the check. formatWindSpeedValue is very simple if it doesn't have to deal with undefined. If I do source.?wind_speed - then I have to handle the case where formatWindSpeedValue accepts an undefined - and decide how to format an undefined.

Reply:
I'd argue most of that logic should live in the formatting function since this is a component returning the view. Also makes it easier to unit test.

Reply:
Done - code is a bit cleaner then as well.